packages/components/src/Combobox/hooks/useComboboxAccessibility.ts

```diff
@@ -15,8 +21,8 @@ export function useComboboxAccessibility(
   wrapperRef: React.RefObject<HTMLDivElement>,
 ): {
   popperRef: React.RefObject<HTMLDivElement>;
-  popperStyles: { [key: string]: React.CSSProperties };
-  attributes: { [key: string]: { [key: string]: string } | undefined };
+  popperStyles: React.CSSProperties;
+  floatingProps: ReturnType<UseInteractionsReturn["getFloatingProps"]>;
```

Comment:
We are also now loosing half of the Combobox when it's near the edge of the browser
![Screenshot 2025-03-25 at 2 24 36 PM](https://github.com/user-attachments/assets/fb43287a-6eda-4960-ba11-3f78e9130ab0)

Whereas before, we'd see the Combobox menu fully:
![Screenshot 2025-03-25 at 2 27 39 PM](https://github.com/user-attachments/assets/3c0cd94b-825e-4cee-b860-e7772b07449e)

Reply:
We can possibly utilize shift? https://floating-ui.com/docs/shift#shift

Reply:
Yup looks like we can call `shift()` in the `middleware` - you can also do `shift({ padding: 8 })` which might be nice to have a bit of a gap

Reply:
@taylorvnoj resolved this in 0c08c33bf